### PR TITLE
Travis now deploys commits from master to PyPI

### PR DESCRIPTION
**Pull request by calcmogul**

### CODE REVIEW

**Comment by virtuald** [comment]
FYI, pypi won't let you overwrite versions

**Comment by calcmogul** [comment]
Will Travis attempt that with this setup? I'm not sure how to test this PR without merging it.

**Comment by virtuald** [comment]
Dunno, how does the deploy functionality for travis work?

**Comment by calcmogul** [comment]
I followed https://docs.travis-ci.com/user/deployment/pypi/. My understanding is it would submit a release to PyPI after a commit on master. I'm not sure whether a Travis build of master would be required for it.

I did this so I could automate package creation for different versions of Python. I'm assuming this is necessary because when I upgraded to Python 3.6.0, it couldn't find wpiformat due to the package being built with Python 3.5.

**Comment by virtuald** [comment]
Oh, your versions are autogenerated. I withdraw my objection. This is probably fine.

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,8 @@
-sudo: required
```
Comment (AustinShalit) [inline .travis.yml]: Yay!

**Pull request merged**